crates/command_silly/src/lib.rs

```diff
@@ -227,6 +227,21 @@ silly_command! {
     }
 }
 
+silly_command! {
+    /// :3
+    fn rickroll("the victim") {
```

Comment:
We use sentence case, capitalize the first letter! (Same goes for other messages of this command.)
```suggestion
    fn rickroll("The victim") {
```